src/collections/events/addo/index.mdx

```diff
@@ -2,9 +2,10 @@
 title: "2020 All Day DevOps"
 date: 2020-11-12
 thumbnail: ./all-day-devops.png
-eurl: https://www.alldaydevops.com/addo-speakers/lee-calcote
+eurl: /community/events/2020-all-day-devops
```

Comment:
I didn't think you were changing these :grimacing: . Do will need to revert these. As they are liked to the buttons on the individual pages. What you can you directly access the links from the graphql query (check workshop for reference) and use that on the cards.

Reply:
The graphql query is used here https://github.com/layer5io/layer5/blob/master/src/templates/event-single.js